### DISCUSSION TITLE
Create folder action

### DISCUSSION THREAD

**Discussion by WillJub** [community/community#147744] [Mobile]
Could you add a create folder option on mobile?
It could be really useful, right now we can only create files, but we can't make a folder structure yet, which can be crucial in some framework contexts.

**Comment by K0979122726** [comment]
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

**Reply by K0979122726** [reply]
ChampionCombat380
